deployment/data-streams/utils/address.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+)
+
+func MaybeFindEthAddress(ab deployment.AddressBook, chain uint64, typ deployment.ContractType) (common.Address, error) {
+	verifierAddrHex, err := deployment.SearchAddressBook(ab, chain, typ)
```

Comment:
verifier?

Reply:
guess it should just be a generic addrHex